openaddr/conform.py

```diff
@@ -1014,6 +1014,19 @@ def conform_smash_case(source_definition):
             if "field_to_remove" in conform[k]:
                 conform[k]["field_to_remove"] = conform[k]["field_to_remove"].lower()
 
+            if "functions" in conform[k]:
+                if type(conform[k]["functions"]) is list:
```

Comment:
These two conditions could be combined, like `if type(conform[k].get('functions')) is list:` or `if "functions" in conform[k] and type(conform[k]["functions"]) is list:`.